Mathlib/Topology/ContinuousOn.lean

```diff
@@ -937,6 +943,11 @@ theorem ContinuousOn.comp {g : β → γ} {f : α → β} {s : Set α} {t : Set
   ContinuousWithinAt.comp (hg _ (h hx)) (hf x hx) h
 #align continuous_on.comp ContinuousOn.comp
 
+@[fun_prop]
+theorem ContinuousOn.comp'' {g : β → γ} {f : α → β} {s : Set α} {t : Set β} (hg : ContinuousOn g t)
```

Comment:
I wrote a small (buggy) script to check that nothing has been lost.  According to the script, this theorem may not have a `fun_prop` before the PR.

Reply:
This one was in the file before, but the arguments may have been in a slightly different order.